### PR TITLE
sanitize the aws label values better

### PR DESCRIPTION
**Pull request by punkle**
sanitize the aws label values better

#### :heavy_check_mark: Checklist

- [x] Added tests for new functionality and regression tests for bug fixes
- [x] Added changeset (run `yarn changeset` in the root)
- [ ] Screenshots of before and after attached (for UI changes)
- [ ] Added or updated documentation (if applicable)

### CODE REVIEW

File: .changeset/tidy-ghosts-care.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@roadiehq/catalog-backend-module-aws': minor
```
Comment (Xantier) [inline .changeset/tidy-ghosts-care.md:2]: patch is enough

**Review by Xantier — DISMISSED** [review]

File: plugins/backend/catalog-backend-module-aws/src/utils/tags.ts [hunk 2]
```diff
@@ -46,7 +46,9 @@ export const labelsFromTags = (tags?: Tag[] | Record<string, string>) => {
       .reduce((acc: Record<string, string>, tag) => {
         if (tag.Key && tag.Value) {
           const key = tag.Key.replaceAll(':', '_').replaceAll('/', '-');
-          acc[key] = tag.Value.replaceAll('/', '-').substring(0, 63);
+          acc[key] = tag.Value.replaceAll('/', '-')
```
Comment (iain-b) [inline plugins/backend/catalog-backend-module-aws/src/utils/tags.ts:49]: I really wonder if we should be adding these as labels at all? These changes materially affect the value and they might not be meaningful / usable without them. We could perhaps use tags? e.g. tags: ['keyA=valueA'] ?

Comment (punkle) [reply plugins/backend/catalog-backend-module-aws/src/utils/tags.ts:49]: Im not sure tbh.

**Review by Xantier — APPROVED** [review]
I'm pro leaving this as labels for now. This will allow more complex entity filtering options if there are need to build such

**Pull request merged**